nephio-ansible-install/roles/kubectl/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+
+---
+- name: Download kubectl binary
+  ansible.builtin.get_url:
+    url: "https://storage.googleapis.com/kubernetes-release/release/v{{ kubectl_version }}/bin/{{ host_os }}/{{ host_arch }}/kubectl"
+    checksum: "{{ kubectl_checksum_binary }}"
+    dest: "{{ tmp_directory }}"
+    mode: 0600
+  tags:
+    - kubectl
+
+- name: Copy kubectl binary to destination directory
+  become: true
+  ansible.builtin.copy:
+    src: "{{ tmp_directory }}/{{ item }}"
+    dest: "{{ bin_directory }}/{{ item }}"
+    mode: "0755"
```

Comment:
Nit: It could be useful to keep mode, owner and group to be parameterized as well for the long term, but OK to keep it hard-coded in the short term.